### ISSUE TITLE
Bottom in readme

### ISSUE THREAD

**Issue by isovector** [robrix/fused-effects#109]
The readme suggests this is a good implementation:

```haskell
instance (Carrier sig m, Member (State s) m) => MTL.MonadState s (Wrapper s m) where
  get = get
  put = put
```

but there is no way that this isn't a bottom :)

**Comment by patrickt** [comment]
Surprise, it isn't bottom! Haskell's scoping rules mean that given these imports
```haskell
import Control.Effect
import Control.Effect.State
import qualified Control.Monad.State as MTL
```

the above instance is resolved as if we had written
```
instance (Carrier sig m, Member (State s) m) => MTL.MonadState s (Wrapper s m) where
  MTL.get = Control.Effect.State.get
  MTL.put = Control.Effect.State.put
```

There is, however, a typo in the head of this instance (`Member` should be over `sig`, not `m`), which I will fix presently.

**Comment by robrix** [comment]
Abstracting a bit from whether or not it’s bottom, it’s certainly confusing; we should probably qualify the rhs definitions (AFAIK you can’t qualify the method names, only the class name).

**Comment by patrickt** [comment]
Fixed with #110.
